src/constitution.md

```diff
@@ -0,0 +1,192 @@
+      - Liaise with SOCIS as required by SOCSSO Policies
+      - Ensure that GCC has a faculty advisor for each semester of operation
+      - Ensure that GCC qualifies for accreditation each semester, as a SOCSSO Club
+      - Ensure that club does not fall into probationary status, as defined by SOCSSO Policies
+
+ ii. **Treasurer**
+
+     Responsibilities:
+
+      - Draft and share financial budgets for each semester of operation
+      - Attend CEPSSC and SOCIS meetings to PDR for funds
+      - Ensure that all expenses (snacks, honorariums, prizes, etc.) are purchased, accounted for in the budget, and reimbursed to the purchaser
+      - Create and present a summary of income and expenditures each semester
+
+
+iii. **Director of Sponsorship**
+
+     Responsibilities:
+
+      - Draft and maintain a sponsorship plan for external organizations which are interested in funding club operations
+      - Correspond with industry representatives
+      - Invite industry representatives to campus to provide guest lectures and workshops
+      - Collaborate with the School of Computer Science Co-op Coordinators to maintain industry interest in Guelph talent
+
+ iv. **Director of Advertising and Outreach**
+
+     Responsibilities:
+
+      - Visit classrooms at the beginning of each semester to promote the Club
+      - Create attractive, informative digital and print advertising for each GCC event
+      - Create and share a plan for advertising, including an outline of different platforms which will be used and any costs that may be incurred by these platforms
+      - Collaborate with SOCIS to ensure access to social media advertising outlets, such as GryphSlack and Instagram
+
+  v. **Director of Events**
+
+     Responsibilities:
+
+      - Coordinate with weekly presenters to review and polish presentation and workshop materials
+      - Host office hours in the Club Space for students who wish to improve their presentation skills
+
+ vi. **Executive at Large**
+
+     Responsibilities:
+
+      - Collaborate with external and internal club groups
+      - Attend community and administrative events where our representation would be beneficial
+      - Support other Executive in decision making, administration, and other tasks
+
+vii. **Committee Chairs**
+
+     Responsibilities:
+
+      - Coordinate a club committee and ensure it is meeting its goals
+
+
+## Committees
+
+Committees may be established by an existing member of the Executive body and volunteered members of the Club to meet a particular goal, or enhance Club operations according to an overarching theme.
+
+Each committee is governed by a corresponding Committee Policy Document, maintained separately from the Constitution.
+
+## Appointments and Elections
+
+Members of the Club Executive are by appointment only, except when the current executive cannot agree on qualified candidates or when there is no qualified candidate for a role.
```

Comment:
Can we define "disagreement"? Just so we don't have a situation where 6 execs think someone is qualified but 1 exec disagrees, and we're unsure if this counts formally as a "disagreement" that must be taken to the following steps.

Reply:
I think we would bring it to a vote if there was any opposition to a proposed new exec member. Basically I'm trying to avoid undue formal process here. I can clarify.

Reply:
Clarified that if exec can't agree on an appointment by simple majority vote, then we will advertise the vacancy and take it to a vote within the executive.